apps/prairielearn/src/sync/fromDisk/courseInstances.ts

```diff
@@ -60,10 +56,12 @@ export async function sync(courseId, courseData) {
   const params = [courseInstanceParams, courseId];
 
   perf.start('sproc:sync_course_instances');
-  const result = await sqldb.callOneRowAsync('sync_course_instances', params);
+  const result = await sqldb.callRow(
+    'sync_course_instances',
+    params,
+    z.record(z.string(), IdSchema),
+  );
```

Comment:
```suggestion
  const result = await sqldb.callRow(
    'sync_course_instances',
    [courseInstanceParams, courseId],
    z.record(z.string(), IdSchema),
  );
```

This also implies removing the params in line 56.

Reply:
Done in 6934d26.